docs/sources/mimir/references/architecture/components/alertmanager.md

```diff
@@ -218,8 +218,10 @@ It's rare to find cases of disagreement in a tenant configuration, as most tenan
 
 1. After identifying and fixing all affected tenant configurations, check the Mimir server logs again to make sure you haven't missed any tenant configurations.
 
-2. You can now enable UTF-8 by setting [`utf8-strict-mode-enabled`]({{< relref "./../../../configure/configuration-parameters#alertmanager" >}}) to `true`.
+2. You can now enable UTF-8 by setting [`utf8-strict-mode-enabled`]({{< relref "./../../../configure/configuration-parameters#alertmanager" >}}) to `true`. You will need to restart Mimir for the changes to take effect.
```

Comment:
```suggestion
1. To enable UTF-8, set [`utf8-strict-mode-enabled`](https://grafana.com/docs/mimir/<MIMIR_VERSION>/configure/configuration-parameters/#alertmanager) to `true`. You need to restart Mimir for the changes to take effect.
```

Reply:
We try to avoid using phrases like "now" and "will" in the docs to always keep them in the present tense. Also, we use the above format for links instead of relrefs now.

Reply:
We just use `1.` for every task step in the markdown. This ensures the step numbers always render properly when we add/remove/update steps.